### PR TITLE
automation cli

### PR DESCRIPTION
**Pull request by Araballi**

### CODE REVIEW

File: .github/workflows/cliAuto-ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Automation Liblibrary CI/CD
```
Comment (rosieks) [inline .github/workflows/cliAuto-ci.yml:1]: Suppose to be _Library_?

Comment (Araballi) [reply .github/workflows/cliAuto-ci.yml]: Fixed

**Comment by NakulSeerpu** [comment]
LGTM

**Review by NakulSeerpu — APPROVED** [review]

**Review by jaceks2106 — CHANGES_REQUESTED** [review]
Generally LGTM with suggestion

**Review by jaceks2106 — APPROVED** [review]

**Pull request merged**